data/pages.yml

```diff
@@ -12,6 +12,8 @@
       url: "index"
     - title: "Your First Ember App"
       url: "first-app"
+    - title: "Glossary"
+      url: "glossary"
     - title: "Core Concepts"
       url: "core-concepts"
```

Comment:
I really like that thought, then as terms are introduced, they could give a quick overview and link to the fully glossary?